funding/commitment_type_negotiation.go

```diff
@@ -86,6 +103,12 @@ func explicitNegotiateCommitmentType(channelType lnwire.ChannelType,
 func implicitNegotiateCommitmentType(local,
 	remote *lnwire.FeatureVector) lnwallet.CommitmentType {
 
+	// If both peers are signalling support for anchor commitments with
+	// zero-fee HTLC transactions, we'll use this type.
+	if hasFeatures(local, remote, lnwire.ScriptEnforcedLeaseOptional) {
```

Comment:
Do we actually need this? Given we can add the explicit chan negotiation as a dependent feature bit for this new lease enforced variant. So this way, we'll never extend this function again, as everything should assume explicit chan negotiation, as there becomes no "default" channel type.

Reply:
Good idea, removed and added explicit channel negotiation as a feature bit dependency.